### PR TITLE
Show the application name in the splash screen #6531

### LINKED ISSUE #6531: Application Name Setting does not apply to splash page
When I set the application name in settings, the browser tab updates with the name on pages except for the splash page.  Would be good it also updated the splash page given that is the intended landing page.

### PR DESCRIPTION
**Pull request by MikeGriniezakis**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

The Application Name setting was not taken into account when rendering the splash screen.
This PR changes the default "Arches" text to use the Application Name setting.

### Issues Solved

Closes #6531 

### Checklist

-   I targeted one of these branches:
    - [x] dev/8.0.x (under development): features, bugfixes not covered below
    - [ ] dev/7.6.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [x] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branch

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/views/main.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def index(request):
             "main_script": "index",
             "active_page": "Home",
             "app_title": settings.APP_TITLE,
+            "app_name": settings.APP_NAME,
```
Comment (chiatt) [inline arches/app/views/main.py]: This doesn't seem to use the project settings, but rather only the Arches settings, so this value remains 'Arches'.

**Review by chiatt — CHANGES_REQUESTED** [review]
Thanks for submitting this @MikeGriniezakis. It seems that the project settings aren't being used in the index view in main.py, but rather the Arches settings, so that will need to be fixed to get this working.

The template changes would also need to be made to the project template so that they are reflected in new projects. You can find that here: arches/arches/install/arches-templates/project_name/templates/index.htm.

File: arches/app/views/main.py (hunk 1 above)
Comment (MikeGriniezakis) [reply arches/app/views/main.py]: I updated the code to use the project settings when getting the APP_NAME in the commit d57bf025b5cd939458cc3a486e502695c3458de9

File: arches/app/views/main.py [hunk 2]
```diff
@@ -16,14 +16,17 @@
...
 """
 
-import re
-import urllib.request, urllib.error, urllib.parse
+import urllib.error
+import urllib.parse
+import urllib.request
 from urllib.parse import urlparse
+
+from django.conf import settings as project_settings
```
Comment (jacobtylerwalls) [inline arches/app/views/main.py:24]: Actually, I don't think we should be importing settings via the standard Django import until we address the behavior difference between dev and prod fleshed out in #11660.

File: arches/app/views/main.py (hunk 1 above)
Comment (jacobtylerwalls) [reply arches/app/views/main.py]: Isn't "Arches" the expected value if there has not been a system settings tile saved for this setting?

Comment (MikeGriniezakis) [reply arches/app/views/main.py]: Upon running the setup_db, the value "Arches" is inserted as the default value into the Application Name setting. With my previous commit, the splash page only changes when the Application Name is changed through the settings.

**Comment by chiatt** [comment]
Hi @MikeGriniezakis - I really appreciate your work on this. Your effort has been really valuable in exposing the importance of addressing #11660 as @jacobtylerwalls has pointed out. In light of this fact, and the fact that the index page is something that generally gets overwritten during project development anyhow, I don't think we should invest more time in this, so I'm going to close this PR and issue. Again, thanks very much for your help.

**Pull request closed without merging**